v0/destinations/active_campaign/transform.js

```diff
@@ -548,11 +548,14 @@ const trackRequestHandler = async (message, category, destination) => {
         "Api-Token": destination.Config.apiKey
       }
     };
+    if (res.response.status !== 201) {
```

Comment:
This can be clubbed with the validation on Ln 527.

Reply:
Updated.